hazelcast/src/main/java/com/hazelcast/map/impl/operation/EntryOperation.java

```diff
@@ -367,29 +377,6 @@ private void unlockOnly(final Object result, String caller, long threadId, long
         updateAndUnlock(null, null, null, caller, threadId, result, now);
     }
 
-    @Override
-    public void onExecutionFailure(Throwable e) {
-        if (offloading) {
```

Comment:
This was always the tricky part, if returnsResponse == false and this means that the caller won't be notified in case of exceptions, that may be retryable. How is this handled in the new code? Don't quite see that anywhere.

Reply:
There is a change in 3.9 that any exception during operation execution is always send no matter what. 

https://github.com/hazelcast/hazelcast/pull/11282